src/main/java/nextstep/subway/path/ui/PathController.java

```diff
@@ -0,0 +1,34 @@
+package nextstep.subway.path.ui;
+
+import nextstep.subway.path.application.PathService;
+import nextstep.subway.line.dto.ShortestPathResponse;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/paths")
+public class PathController {
+    private final PathService pathService;
+
+    public PathController(PathService pathService) {
+        this.pathService = pathService;
+    }
+
+    @GetMapping
+    public ResponseEntity findShortestPath(
```

Comment:
```suggestion
    public ResponseEntity<ShortestPathResponse> findShortestPath(
```

가급적이면 제네릭 형식을 맞춰주세요.
확인도 편하고 코드 정합성에도 도움이 됩니다.